server/eclipse-project/src/test/java/us/freeandfair/corla/controllers/ContestCounterTest.java

```diff
@@ -0,0 +1,61 @@
+package us.freeandfair.corla.controllers;
+
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.HashMap;
+import java.util.HashSet;
+
+import org.testng.annotations.Test;
+import org.testng.Assert;
+
+import us.freeandfair.corla.controller.ContestCounter;
+
+public class ContestCounterTest {
+
+
+  private ContestCounterTest (){};
+
+  @Test()
+  public void testAccumulateVoteTotals(){
+    List<Map<String,Integer>> voteTotals = exampleVoteTotals();
+
+    Map<String,Integer> results = ContestCounter.accumulateVoteTotals(voteTotals);
+    Assert.assertEquals((int)results.size(), 2);
+    Assert.assertEquals((int)results.get("Joe Frasier"), 18);
+    Assert.assertEquals((int)results.get("Muhammed Ali"), 20);
```

Comment:
Is the `(int)` cast needed?

Reply:
There's also `assertEquals(Object, Object)`, and you'll hear about it 🔊 from TestNG.